testdata/source/amazon/D56-5204779-4181560.json

```diff
@@ -54,7 +54,7 @@
         }
     ],
     "pretax_adjustments": [],
-    "tax": [],
+    "tax": null,
```

Comment:
I changed this, since otherwise the code (also the original?!) throws an exception here:
https://github.com/jbms/beancount-import/blob/dca340694ac60dd8ee7529cc4740b09fe729dbc2/beancount_import/source/amazon.py#L390

so I set `invoice.tax=None` instead of `=[]` for digital invoices

Reply:
Can we change the code instead, so all the fields in the json follow the same convention (`field = []`).

Reply:
makes sense, I changed the code in `amazon.py` to handle `[]`